main.tf

```diff
@@ -150,10 +164,58 @@ resource "kubernetes_deployment" "wandb" {
             }
           }
         }
+
+        container {
+          name = "sidecar"
+          image = "busybox"
+
+          volume_mount {
+            name = "gorilla"
+            mount_path = "/var/log/${local.gorilla}"
+            sub_path = local.gorilla
+          }
+
+          volume_mount {
+            name = "local" 
+            mount_path = "/var/log/${local.local}"
+            sub_path = local.local
+          }
+
+          args = [ "/bin/sh", "-c", "tail -n+1 -f /var/log/gorilla.log" ]
+        }
+
         volume {
           name = local.app_name
           config_map {
             name     = kubernetes_config_map.config_map.metadata[0].name
+            items {
+              key = "server_ca.pem"
+              path = "server_ca.pem"
+            }
+            optional = true
+          }
+        }
+
+        volume {
+          name = "gorilla"
+          config_map {
+            name = kubernetes_config_map.config_map.metadata[0].name
+            items {
+              key = "gorilla.log"
+              path = "gorilla.log"
+            }
+            optional = true
+          }
+        }
+
+        volume {
+          name = "local"
+          config_map {
+            name = kubernetes_config_map.config_map.metadata[0].name
+            items {
+              key = "local.log"
+              path = "local.log"
+            }
```

Comment:
Why are we using a `config_map` here?  This should just be a ReadManyWriteOnce volume and the names of these logs should be piped through the using the `local` vars.